src/bots/filler.ts

```diff
@@ -2269,6 +2317,74 @@ export class FillerBot extends TxThreaded implements Bot {
 		return true;
 	}
 
+	public findTriggerableNodesWithMakers(
+		dlob: DLOB,
+		marketIndex: number,
+		slot: number,
+		oraclePriceData: OraclePriceData,
+		marketType: MarketType,
+		stateAccount: StateAccount
+	): NodeToTriggerWithMakers[] {
+		const baseTriggerable = dlob.findNodesToTrigger(
+			marketIndex,
+			slot,
+			oraclePriceData.price,
+			marketType,
+			stateAccount
+		);
+
+		const triggerWithMaker: NodeToTriggerWithMakers[] = [];
+
+		for (const nodeObj of baseTriggerable) {
+			const order = nodeObj.node.order;
+			let isFillable = false;
+
+			if (
+				isVariant(order.orderType, 'market') ||
+				isVariant(order.orderType, 'triggerMarket')
+			) {
+				isFillable = true;
+			} else if (isVariant(order.orderType, 'triggerLimit')) {
+				if (
+					isVariant(order.triggerCondition, 'triggeredAbove') ||
+					isVariant(order.triggerCondition, 'above')
+				) {
+					isFillable = order.price.lte(oraclePriceData.price);
```

Comment:
this check doesn't work if the dlob is trading far away from oracle. I'd just always attempt to fill instead of adding a `isFillable`